src/usage.rs

```diff
@@ -0,0 +1,1226 @@
+    DirectoryReadOnly,
+    /// View your street addresses
+    UserAddressesRead,
+    /// See and download your exact date of birth
+    UserBirthdayRead,
+    /// See and download all of your Google Account email addresses
+    UserEmailsRead,
+    /// See your gender
+    UserGenderRead,
+    /// Manage your product listings for Google Manufacturer Center
+    ManufacturerCenter,
+    /// See and download contact info automatically saved in your "Other contacts"
+    ContactsOtherReadOnly,
+    /// Submit data to Google for indexing
+    Indexing,
+    /// See and delete your domain's G Suite alerts, and send alert feedback
+    AppsAlerts,
+    /// Manage users on your domain
+    AppsOrder,
+    /// Manage users on your domain
+    AppsOrderReadOnly,
+    /// Upload messages to any Google group in your domain
+    AppsGroupsMigration,
+    /// View and manage the settings of a G Suite group
+    AppsGroupsSettings,
+    /// View your eDiscovery data
+    EDiscoveryReadOnly,
+    /// Manage your eDiscovery data
+    EDiscovery,
+    /// Create, edit, organize, and delete all your tasks
+    Tasks,
+    /// View your tasks
+    TasksReadOnly,
+    /// See all your Google Sheets spreadsheets
+    SpreadsheetsReadOnly,
+    /// See, edit, create, and delete all your Google Slides presentations
+    Presentations,
+    /// See all your Google Slides presentations
+    PresentationsReadOnly,
+    /// See and download all your Google Drive files
+    DriveReadOnly,
+    /// See, edit, create, and delete only the specific Google Drive files you use with this app
+    DriveFile,
+    /// Manage your new site verifications with Google
+    SiteVerificationVerifyOnly,
+    /// Manage the list of sites and domains you control
+    SiteVerification,
+    /// View Search Console data for your verified sites
+    WebmastersReadOnly,
+    /// View and manage Search Console data for your verified sites
+    Webmasters,
+    /// View and manage your Google Play Developer account
+    AndroidPublisher,
+    /// See, create, and delete its own configuration data in your Google Drive
+    DriveAppData,
+    /// Create, edit, and delete your Google Play Games activity
+    Games,
+    /// Manage corporate Android devices
+    AndroidEnterprise,
+    /// Modify your Google Apps Script scripts' behavior
+    DriveScripts,
+    /// View the photos, videos and albums in your Google Photos
+    DrivePhotosReadOnly,
+    /// See information about your Google Drive files
+    DriveMetaDataReadOnly,
+    /// View and manage metadata of files in your Google Drive
+    DriveMetaData,
+    /// See all your Google Docs documents
+    DocumentsReadOnly,
+    /// See your primary Google Account email address
+    Email,
+    /// Associate you with your personal info on Google
+    OpenId,
+    /// See your personal info, including any personal info you've made publicly available
+    Profile,
+}
+
+impl Usage {
+    pub(crate) fn as_str(&self) -> String {
```

Comment:
Jus some thoughts:
i was lately very frustrated by libraries hiding lots of there functionality when i tried to do some (i have to admit not really intended) "haching" for the mensa app authentication.
So there is a consideration to provide these string representation to the user. but that of corse has some stability implication as it then is part of the public api.
This may be also a candidate for the display trait (which will automatically provide you with a ToString implementation. (you could simply use ith method inside a Display impl)